### DISCUSSION TITLE
Change station folder name without media rescan

### DISCUSSION THREAD

**Discussion by RM-FM** [AzuraCast/AzuraCast#6744] [General]
Hi,

I was wondering whether there is a way to rename the station name folder in volume azuracast_station_data without triggering a reset/rescan of the media files? I guess this can be achieve only by changing the path in the database. Anyone managed to do that?

**Comment by BusterNeece** [comment]
@RM-FM The easiest way to accomplish this is to visit Administration > Storage Locations, then edit the path of the storage location there. It'll keep its association with all of the stations it's already associated with, and will also keep all of its scanned media, since the paths for scanned media are relative to the base of the directory, not absolute to the full filesystem.

**Reply by Moonbase59** [reply]
That is valuable info! And well thought out, kudos.
